autolink/postlink/activityLinker.js

```diff
@@ -1,23 +1,52 @@
 // @ts-check
-var path = require('./path');
+var path = require("./path");
 var fs = require("fs");
-var { warnn, logn, infon, debugn } = require("./log");
+var { errorn, warnn, logn, infon, debugn } = require("./log");
 
 class ActivityLinker {
   constructor() {
     this.activityPath = path.mainActivityJava;
+    this.extendNavigationActivity = false;
+    this.removeGetMainComponentName = false;
```

Comment:
I added these status variables to keep tabs on some of the necessary actions that could fail during the process

Reply:
I saw similar variables in other files with `Success` as a suffix.
Should we rename these ones?

Reply:
Yes, @ignacioola commented the same, I used one criteria first and then ditched it on the rest 😓